homeassistant/components/samsungtv/manifest.json

```diff
@@ -4,15 +4,18 @@
   "documentation": "https://www.home-assistant.io/integrations/samsungtv",
   "requirements": [
     "samsungctl[websocket]==0.7.1",
-    "samsungtvws==1.6.0"
+    "samsungtvws[websocket]==1.6.0"
   ],
   "ssdp": [
     {
       "st": "urn:samsung.com:device:RemoteControlReceiver:1"
     }
   ],
+  "zeroconf": [
+    "_airplay._tcp.local."
```

Comment:
```suggestion
    {"type":"_airplay._tcp.local.","manufacturer":"samsung*"}
```

Reply:
Requires https://github.com/home-assistant/core/pull/48810

Reply:
If you rebase, this should now work

Reply:
Create PR to fix hassfest script: #49006